prov/efa/src/rdm/rxr_ep.c

```diff
@@ -2484,6 +2523,51 @@ void rxr_ep_handle_misc_shm_completion(struct rxr_ep *ep,
 	}
 }
 
+/**
+ * @brief handle RX completion due to FI_REMOTE_CQ_DATA via RECV_RDMA_WITH_IMM
+ *
+ * This function handles hardware-assisted RDMA writes with immediate data at
+ * remote endpoint.  These do not have a packet context, nor do they have a
+ * connid available.
+ *
+ * @param[in]		ep		endpoint
+ * @param[in]		cq_entry	CQ entry from shm provider
```

Comment:
There is no `cq_entry` and `src_addr` in this function

Reply:
Thank you